zeroex/orderwatch/order_watcher.go

```diff
@@ -29,9 +29,14 @@ const (
 	// were not caught by the event watcher process.
 	minCleanupInterval = 1 * time.Hour
 
-	// minRemovedCheckInterval specifies the minimum amount of time between checks
-	// on whether to remove orders flaggged for removal from the DB
-	minRemovedCheckInterval = 5 * time.Minute
+	// maxDeleteInterval specifies the maximum amount of time between calls to permanentlyDeleteStaleRemovedOrders
+	maxDeleteInterval = 5 * time.Minute
+
+	// countInterval specifies the amount of time between calls to db.CountOrders. If the count is higher than a threshold, then permanentlyDeleteStaleRemovedOrders will be called, so this is the minimum interval between calls to permanentlyDeleteStaleRemovedOrders.
+	countInterval = 5 * time.Second
```

Comment:
I think naming this `countInterval` is not specific enough. Keep in mind that this constant is in scope for the entire `orderwatch` package. How about `checkDatabaseUtilizationThresholdInterval`?